src/app/base/components/node/NodeTestsTable/NodeTestsTable.tsx

```diff
@@ -174,6 +174,7 @@ const NodeTestsTable = ({ node, scriptResults }: Props): JSX.Element => {
   return (
     <>
       <MainTable
+        aria-label="Test results"
```

Comment:
It doesn't look like you ended up using this label (as far as I can see).

Reply:
I will, once this is in the dev snap and can be used in tests (right now if I tried `cy.findByRole("grid", { name: "Test results" }` this would have failed.